sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -21,6 +21,9 @@ import java.beans.Introspector
 import java.util.Properties
 import java.util.concurrent.atomic.AtomicReference
 
+import org.apache.spark.sql.catalyst.analysis.FunctionRegistry.{expression, FunctionBuilder}
```

Comment:
sort the imports - also I wouldn't import "expression" here, since it too general of a keyword.